client/my-sites/subscribers/components/subscribers-page/subscribers-page-context.tsx

```diff
@@ -141,6 +142,7 @@ export const SubscribersPageProvider = ( {
 				setShowAddSubscribersModal,
 				addSubscribersCallback,
 				siteId,
+				isFetching: subscribersQueryResult.isFetching,
```

Comment:
The loading placeholders are being displayed multiple times as `isFetching` is true every time that I react-query refreshes the data. I think `isLoading` could solve this issue.